01-Login/src/app/home/home.component.ts

```diff
@@ -1,13 +1,18 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 import { environment } from 'src/environments/environment';
 
 @Component({
-  selector: 'home',
+  selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css']
 })
-export class HomeComponent {
-  title = 'home';
+export class HomeComponent implements OnInit {
 
   public appId: string = environment.passageAppId;
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
you don't need ngOnInit if its just empty (just also remove the `implements OnInit` and the TypeScript gods will be appeased as well)

Reply:
Excellent -- thanks Kevin.